### ISSUE TITLE
Standard field names: documentation?

### ISSUE THREAD

**Issue by rubdos** [invoice-x/invoice2data#218]
Should there be a documentation file with all the fields, line fields and table fields that are used (including the optional ones)? This should be useful for compatibility.

**Comment by m3nu** [comment]
Good idea. We already have a [template developer tutorial](https://github.com/invoice-x/invoice2data/blob/master/TUTORIAL.rst), which is related.

Maybe it's also time for a ReadTheDocs site with further info.
